yamls/loadbalancer-values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   repository: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon/aws-load-balancer-controller
-  tag: v2.1.2
+  tag: v2.4.1
   pullPolicy: IfNotPresent
 
```

Comment:
If we decide the consumer to specify Helm chart version, we should not hardcode this either.